flipperdevices/flipperzero-good-faps
# Picopass save written key

# What's new

- When there is no user dictionary and the key of a card is changed to something the user input, save that value to a newly created user dictionary.  We believe we've seen people get into a bad state (key all 0's) when they don't know what they're doing.

# Verification 

- Backup/delete user dictionary
- Read a standard keyed iClass
- Write key (defaults to all 0's)
- See user dictionary with all 0's key was created
- Try to read card again, see it was successful using all 0's key from user dictionary.

# Checklist (For Reviewer)
